composer.json

```diff
@@ -14,22 +14,23 @@
         }
     ],
     "require": {
-        "php": "^7.1",
+        "php": "^7.2",
         "gmostafa/php-graphql-client": "^1.4",
         "phpdocumentor/reflection-docblock": "^5.1",
-        "symfony/property-info": "4.4.*",
+        "symfony/property-access": "^4.3",
+        "symfony/property-info": "^4.3",
         "symfony/serializer": "^4.3"
```

Comment:
Dit is nog wel een dingetje, met deze restricties gaat het dus niet werken in een symfony 5 applicatie. Ik denk dat dit "^4.3,^5.0" moet worden, hoewel we natuurlijk wel moeten testen dat het inderdaad werkt met de sf 5(.0) versies van die components dan.
Had ik niet eerder over nagedacht, beetje tegenintuitief dat je je library dus eigenlijk slechter te gebruiken maakt in een symfony applicatie door symfony components als dependency te hebben.
We kunnen hier evt. ook een apart issue van maken, maar ik denk wel dat het belangrijk is.

Reply:
- [ ] testen sf 3 - sf 5